src/pytest/__init__.py

```diff
@@ -116,7 +115,6 @@
     "Mark",
     "MarkDecorator",
     "MarkGenerator",
-    "Metafunc",
```

Comment:
Is this intentional? If so this should count as a breaking change, I think.

Reply:
Good catch, refactoring tool mistake